packages/core/src/physics/PhysicsManager.ts

```diff
@@ -138,6 +140,17 @@ export class PhysicsManager {
 
   constructor(engine: Engine) {
     this._engine = engine;
+  }
+
+  /**
+   * initialize PhysicsManager.
+   * @param physics - Physics Engine
+   */
+  initialize(physics: IPhysics): void {
+    if (PhysicsManager._nativePhysics) {
```

Comment:
This condition should be:
```typescript
if (this._initialized) return;
```
Because `initialize` is a method of  instance, not a static method of class. If there are two instances of `PhysicsManager`, the second will never initialize if the first have been initialized.

Reply:
done